base/compiler/abstractinterpretation.jl

```diff
@@ -568,6 +568,7 @@ function maybe_get_const_prop_profitable(interp::AbstractInterpreter, result::Me
         return nothing
     end
     method = match.method
+    method.constprop == 0x02 && return nothing
```

Comment:
```suggestion
    if method.constprop == 0x02
        add_remark!(interp, sv, "[constprop] Disabled by method parameter")
        return nothing
    end
```

I will try to setup some UI on Cthulhu to show these remarks later.